.github/container/test-t5x.sh

```diff
@@ -206,3 +234,29 @@ ENABLE_TE=$ENABLE_TE python -m t5x.train \
     $ADDITIONAL_ARGS \
     $([[ $MULTIPROCESS != 0 ]] && echo --multiprocess_gpu)
 echo "Output at ${OUTPUT}"
+
+if [[ "$ENABLE_FMHA" -eq "1" ]]; then 
+    ## Check if fmha instructions are present in the HLO dumped file or not.
+    fmha_regex="fmha[-bmm]?[-scale]?[-bias]?[-mask]?[-softmax]?[-dropout]?[-bmm]?[-backward]?*"
+    result=$(grep -irlnE "$fmha_regex" "${HLO_DIR}/"*.txt)
+
+    if [[ $SAVE_HLO -eq 0 ]]; then
+	    rm -rf $HLO_DIR
+ 	    echo "Removed dumped HLO directory!"
+    fi
+
+    if [ -z "$result" ]; then
+        echo "E: No FMHA instructions were found in the hlo files!"
+	    exit 1
+    else
+	    echo -e "Found FMHA instructions in the following HLO files: \n $result" | while read -r line; do
```

Comment:
Maybe I'm missing something, but why the `while`? `echo -e "$multiline_string_var" should preserve newlines

Reply:
Incorporated review comments